File: AuctionServer/AuctionServer.Domain/Entities/Bid.cs
namespace AuctionServer.Domain.Entities;

public class Bid
{
    public Guid Id { get; set; }
    public float Amount { get; set; }
    public DateTime Timestamp { get; set; }
    public bool isWinner { get; set; }
    
    public Guid? BidderId { get; set; }
    public User? Bidder { get; set; }
    
    public Guid? AuctionItemId { get; set; }
    public AuctionItem? AuctionItem { get; set; }
}

File: AuctionClient/src/types/Bid.ts
export interface Bid {
  id: string;
  amount: number;
  timestamp: string;
  isWinner: boolean;
  bidderId: string;
  auctionItemId: string;
}
export interface BidCreation {
  amount: number;
  bidderId: string;
  auctionItemId: string;
}
